react-accordion-component/my-app/src/Accordion.js

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
+
+export default function Accordion({ topics }) {
+  const [activeIndex, setActiveIndex] = useState();
+  function handleOnClick(index) {
+    index === activeIndex ? setActiveIndex() : setActiveIndex(index)
+  }
+
+  return <Panel topics={topics} onClick={handleOnClick} currentIndex={activeIndex}/>
+}
+
+function Panel({ topics, onClick, currentIndex }) {
+  const topicsLists = topics.map((topic, index) => (
```

Comment:
There's no `key` prop in here. Check the console for the error message.
Fix that, please.